Note about computing rids (#30168)
dotnet/docs: docs/core/rid-catalog.md

They're used by .NET packages to represent platform-specific assets in NuGet packages. The following values are examples of RIDs: `linux-x64`, `ubuntu.14.04-x64`, `win7-x64`, or `osx.10.12-x64`.
For the packages with native dependencies, the RID designates on which platforms the package can be restored.

> A single RID can be set in the [<RuntimeIdentifier>](project-sdk/msbuild-props.md#runtimeidentifier) element of your project file. Multiple RIDs can be defined as a semicolon-delimited list in the project file's [<RuntimeIdentifiers>](project-sdk/msbuild-props.md#runtimeidentifiers) element. They're also used via the `--runtime` option with the following [.NET CLI commands](./tools/index.md):

Reviewer: Code style won't be rendered but it keeps this from being interpreted as a html element. You could escape the open angle bracket instead.

```suggestion
A single RID can be set in the [`<RuntimeIdentifier>`](project-sdk/msbuild-props.md#runtimeidentifier) element of your project file. Multiple RIDs can be defined as a semicolon-delimited list in the project file's [<RuntimeIdentifiers>](project-sdk/msbuild-props.md#runtimeidentifiers) element. They're also used via the `--runtime` option with the following [.NET CLI commands](./tools/index.md):
```

- `[version]` is the operating system version in the form of a dot-separated (`.`) version number. For example, `15.10`.

>   The version **shouldn't** be a marketing version, as they often represent multiple discrete versions of the operating system with varying platform API surface area.

Reviewer: Changing versions to singular makes it out of sync with the following "they"

```suggestion
  The version **shouldn't** be a marketing version, as a marketing version often represents multiple discrete versions of the operating system with varying platform API surface area.
```

Author: ```suggestion
  The version **shouldn't** be a marketing version, as marketing versions often represent multiple discrete versions of the operating system with varying platform API surface area.
```
